modules/aws-outposts-load-balancer-clb-fix-subnet.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-load-balancer-clb-fix-subnet_{context}"]
+= Ensuring AWS load balancers use the correct subnet
+
+When you extend an AWS VPC cluster to an Outpost, the subnet discovery by the cloud controller manager might cause new AWS Classic Load Balancers and Network Load Balancers to be added to the Outpost availability zone.
+Because AWS Outposts does not support these types of load balancers, you must prevent them from being added to the Outpost subnet by adding an annotation with the cloud-based subnets.
+
+[NOTE]
+====
+Manually ensuring that new Classic Load Balancers and Network Load Balancers use the correct subnets is a workaround for clusters installed with {product-title} {product-version}.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+//Needs to be updated for NLB too. Also, might be able to remove the deployment part and refer to user workloads as a pre-req.
+. Configure the Classic Load Balancer `deployment` resource:
++
+.Example deployment configuration
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <application_name>
+  namespace: <application_namespace>
+spec:
+  selector:
+    matchLabels:
+      app: <application_name>
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: <application_name>
+    spec:
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: <application_name>
+          ports:
+            - containerPort: 8080
+----
+
+. Create the `deployment` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_deployment>.yaml
+----
```

Comment:
Marco proposed dropping this and adding "you have created a user workload in the Outpost" as a pre-req